components/automate-deployment/pkg/server/api_token_test.go

```diff
@@ -134,7 +134,7 @@ func TestGenerateAdminToken(t *testing.T) {
 		}
 
 		req := &api.GenerateAdminTokenRequest{
-			Description: testDescription,
+			Description: testName,
```

Comment:
Let's rename this Description field too

Reply:
Lemme see how big of a change that'll end up being. If big I'll do it in a follow up.

Reply:
Looks like it's only used in one place, I'll fix it up here.